systemvm/patches/debian/config/opt/cloud/bin/configure.py

```diff
@@ -945,8 +945,10 @@ def main(argv):
 
         if process_file in ["cmd_line.json", "vm_password.json"]:
             logging.debug("Configuring vmpassword")
-            password = CsPassword("vmpassword", config)
-            password.process()
+            # TODO fix the line below
+            # password = CsPassword("vmpassword", config)
```

Comment:
I don't feel confident this will be backwards compatible. We are now not configuring the vm with a password.